groovy/groovy.editor/test/qa-functional/src/org/netbeans/test/groovy/cc/testCCMixed.java

```diff
@@ -196,9 +196,9 @@ public void ImportPackage() {
         type(file, "\n public static int testInt(){\n return 0;");
         new EventTool().waitNoEvent(1000);
         file.save();
-        createGroovyFile(TEST_BASE_NAME + (name_iterator - 1), "Groovy Class", "EEE", "org.netbens.groovy");
+        createGroovyFile(TEST_BASE_NAME + (name_iterator - 1), "Groovy Class", "EEE", "org.netbeans.groovy");
```

Comment:
You're changing a test here. Did you test it afterwards?

Reply:
netbens fixed at 159fcdc8